sample/Tests/browserstack.ios.yml

```diff
@@ -26,7 +26,7 @@ source: pytest-browserstack:sample-sdk:v1.0
 # Set `app` to define the app that is to be used for testing.
 # It can either take the id of any uploaded app or the path of the app directly.
 #app: ./WikipediaSample.apk
-app: ./Payload.ipa #For running local tests
+app: ../build/output/iOS/IPA/Payload.ipa #For running local tests
```

Comment:
@nattb8 Im sure it's not `Payload.ipa` but I don't know what it is, need to change